pkg/model/requests_processing_test.go

```diff
@@ -68,24 +69,71 @@ func testRequestsProcessing(t *testing.T, describe spec.G, it spec.S) {
 			assert.Equal(t, simulator.StockName("RequestsProcessing [99]"), subject.Name())
 		})
 	})
-
 	describe("Add()", func() {
-		var request simulator.Entity
+		var request RequestEntity
 
-		it.Before(func() {
-			bufferStock := NewRequestsRoutingStock(envFake, NewReplicasActiveStock(), nil)
-			request = NewRequestEntity(envFake, bufferStock, RequestConfig{CPUTimeMillis: 200, IOTimeMillis: 200, Timeout: 3 * time.Second})
-			subject.Add(request)
+		describe("there is no free cpu resource", func() {
+			it.Before(func() {
+				*rawSubject.occupiedCPUCapacityMillisPerSecond = *rawSubject.totalCPUCapacityMillisPerSecond
+				routingStock := NewRequestsRoutingStock(envFake, NewReplicasActiveStock(), nil)
+				request = NewRequestEntity(envFake, routingStock, RequestConfig{CPUTimeMillis: 200, IOTimeMillis: 200, Timeout: 3 * time.Second})
+				subject.Add(request)
+			})
+			describe("scheduling processing", func() {
+				it("schedules a movement from RequestsProcessing to RequestsFailed", func() {
+					assert.Equal(t, simulator.StockName("RequestsFailed"), envFake.Movements[0].To().Name())
+				})
+			})
+
+			it("there is 1 entity in processingStock", func() {
```

Comment:
Can we assert the name or something of that 1 entity? Then it's more clear why 1 and what we expect there.

Reply:
The purpose of this test to verify that there is only one entity in stock as I added only one request. Are you sure that I should check request parameters in this test?

Reply:
Ok